input/docs/handbook/commands/index.md

```diff
@@ -76,7 +74,7 @@ LoadUsers = ReactiveCommand.CreateFromTask(LoadUsersAsync);
 // LoadUsers Observable, update our OAPH and notify the UI 
 // that the value of Users property has changed.
 _users = LoadUsers.ToProperty(
-    this, x => x.Users, RxApp.MainThreadScheduler
+    this, x => x.Users, scheduler: RxApp.MainThreadScheduler);
```

Comment:
Looks like the syntax became invalid, now we have two `);` here.

Reply:
Thank you for pointing out my mistake.